### PR TITLE
add a README with a high-level description

### PR DESCRIPTION
**Pull request by marten-seemann**
Depends on #10.

Not sure if my text is really understandable, but at least it's something. Would be happy about suggestions how to improve it.

### CODE REVIEW

**Review by willscott — COMMENTED** [review]
consider following the [standard readme](https://github.com/RichardLitt/standard-readme) format? we do that in many of the ipfs side repos at least?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+This repository contains GitHub Actions workflows used by various IPLD repositories.
+By storing them in a central place (here), and distributing them in an automated way, we solve multiple problems:
+1. Consistency: Every participating repository uses the same workflow, ensuring that our code adheres to the same coding standards and is properly tested.
+2. Maintainability: Workflows change over time. We need to be able to make changes without manually updating dozens of repositories.
+
+## Technical Details
+
+This repository currently defines two workflows for Go repositories:
+* [go-check](https://github.com/ipld/.github/blob/master/workflow-templates/go-check.yml): Performs checks that ensure that the code adheres to common coding styles.
```
Comment (mvdan) [inline README.md:11]: all these links could be relative, like `[go-check](workflow-templates/go-check.yml)`, reducing duplication and rendering well even if this git repository gets forked or hosted elsewhere.

Comment (mvdan) [reply README.md:11]: Also, I think "coding styles" is too narrow; we already run vet, which finds bugs via static analysis, and we'll likely add others.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+* [go-test](https://github.com/ipld/.github/blob/master/workflow-templates/go-test.yml): Runs unit tests, using different compiler versions and operating systems.
```
Comment (mvdan) [inline README.md:12]: nit: `go test` may be about more than unit tests :) I think you should just say "runs all tests".

**Comment by marten-seemann** [comment]
> consider following the standard readme format? we do that in many of the ipfs side repos at least?

@willscott I'm not sure how the standard readme would look for this repo. What exactly would you change?

File: README.md (hunk 1 above)
Comment (marten-seemann) [reply README.md]: @mvdan Any suggestion how to say this?

**Comment by willscott** [comment]
> > consider following the standard readme format? we do that in many of the ipfs side repos at least?
> 
> @willscott I'm not sure how the standard readme would look for this repo. What exactly would you change?

* add a license section at the end :) the go-ipfs one is good.
* maybe add a usage section for how a repo can opt-in?

File: README.md (hunk 1 above)
Comment (marten-seemann) [reply README.md]: @mvdan Can I ask for your help here?

Comment (mvdan) [reply README.md]: Sorry that I missed your earlier comment. How about:

> Performs static analysis, style, and formatting checks to help improve the code quality.

Comment (marten-seemann) [reply README.md]: Sounds good!

File: README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+This repository contains GitHub Actions workflows used by various IPLD repositories.
+By storing them in a central place (here), and distributing them in an automated way, we solve multiple problems:
+1. Consistency: Every participating repository uses the same workflow, ensuring that our code adheres to the same coding standards and is properly tested.
+2. Maintainability: Workflows change over time. We need to be able to make changes without manually updating dozens of repositories.
+
+## Technical Details
+
+This repository currently defines two workflows for Go repositories:
+* [go-check](workflow-templates/go-check.yml): Performs static analysis, style, and formatting checks to help improve the code quality.
+* [go-test](workflow-templates/go-test.yml): Runs unit tests, using different compiler versions and operating systems.
```
Comment (mvdan) [inline README.md:12]: s/unit tests/all tests/

**Review by mvdan — APPROVED** [review]

**Pull request merged**